daprdocs/content/en/developing-applications/building-blocks/state-management/howto-outbox.md

```diff
@@ -106,6 +108,73 @@ spec:
     value: "newOrder"
 ```
 
+### Shape the outbox pattern message
+
+You can override the outbox pattern message saved to the database during the transaction by setting a different message. This is done via a projected transaction payload, which is ignored in the database, but used as the outbox pattern message published to the user topic.
```

Comment:
This feature overrides the message that gets published to the pub/sub broker, not the database